cumulus_etl/loaders/i2b2/transform.py

```diff
@@ -347,4 +352,6 @@ def make_concept(code: str, system: str | None, display: str = None) -> dict:
     coding = {"code": code, "system": system}
     if display is not None:
         coding["display"] = display
+    elif system in ICD_CODE_DICT:
+        coding["display"] = ICD_CODE_DICT[system].get(code, 'Display not found')
```

Comment:
This seems intentional, but is `Display not found` the text we want to show up in graphs? Vs leaving as `None` and letting the library or dashboard put a string in (my gut instinct)? Or even something vaguer/less-techy like `Unknown`?

Reply:
i can live with Unknown. 

There's a 'no one wants to to own this piece' kind of argument going on here - we really want this to come from the data source directly? and if it weren't this pre-FHIR legacy thing we'd be going back to them? But an 'Unknown' string :could: be easily replaced in either of the other layers if we needed it to.